File: Math/codeforces_1845B.py

# Problem link ===>> https://codeforces.com/problemset/problem/1845/B

def solve():
  t = int(input())
  for _ in range(t):
    xA, yA, xB, yB, xC, yC = map(int, input().split())
    distAB = abs(xA - xB) + abs(yA - yB)
    distAC = abs(xA - xC) + abs(yA - yC)
    distBC = abs(xB - xC) + abs(yB - yC)
    minDist = min(distAB, min(distAC, distBC))
    if minDist == distAB:
      print("1", distAC + distBC - distAB)
    elif minDist == distAC:
      print("2", distAB + distBC - distAC)
    else:
      print("3", distAB + distAC - distBC)

if __name__ == "__main__":
  solve()

File: Math/codeforces_1845B.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/1845/B
// submission link ===>> https://codeforces.com/contest/1845/submission/214097714

#include <iostream>
#include <vector>

using namespace std;

int main()
{
    int t;
    cin >> t;
    for (int i = 0; i < t; i++)
    {
        int xA, yA, xB, yB, xC, yC;
        cin >> xA >> yA >> xB >> yB >> xC >> yC;
        int distAB = abs(xA - xB) + abs(yA - yB);
        int distAC = abs(xA - xC) + abs(yA - yC);
        int distBC = abs(xB - xC) + abs(yB - yC);
        int minDist = min(distAB, min(distAC, distBC));
        if (minDist == distAB)
        {
            cout << "1 " << distAC + distBC - distAB << endl;
        }
        else if (minDist == distAC)
        {
            cout << "2 " << distAB + distBC - distAC << endl;
        }
        else
        {
            cout << "3 " << distAB + distAC - distBC << endl;
        }
    }
    return 0;
}
